src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -237,24 +244,76 @@ class SingleActiveFIMap extends React.Component<
     };
     const namePaths =
       plan.jurisdiction_name_path instanceof Array ? plan.jurisdiction_name_path : [];
-    const pages = namePaths.map(namePath =>
+    const pages = namePaths.map(namePath => {
       // return a page object for each name path
-      ({
+      return {
         label: namePath,
         url: '',
-      })
-    );
+      };
+    });
     breadCrumbProps.pages = [homePage, basePage, ...pages, secondLastPage];
+    const statusBadge =
+      plan && plan.plan_status === PlanStatus.ACTIVE ? (
+        <Badge color="warning" pill={true}>
+          Active
+        </Badge>
+      ) : (
+        <Badge color="success" pill={true}>
+          Complete
+        </Badge>
+      );
+    const markCompleteButton =
+      plan && plan.plan_status === PlanStatus.ACTIVE ? (
+        <Button color="primary" size="lg" block={true}>
+          Mark as Complete
+        </Button>
+      ) : null;
+
+    const cols: string[] = Object.keys(plan);
```

Comment:
Can we either rename this to a more descriptive name, or add a comment on what is for?